cypress/e2e/API/api.cy.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="Cypress" />
+
+//API Test Suit
+describe('API Tests', () => {
+
+    it('Recupera clientes com sucesso', () => {
+        //Realiza uma request para a p�gina e verifica se a resposta � 200
```

Comment:
Que tal em vez de um comentário, pasar tal informação na descrição do bloco `it`?

Reply:
A propósito, cuidado com caracteres estranhos, como `�`.